webpack.config.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+    publicPath: '/',
+  },
+  optimization: {
```

Comment:
- [ ] Kindly set up webpack mode for development or production to avoid the warnings I'm getting when I run the project.

![webpack](https://github.com/Bupilipili/To-do-list/assets/53124548/993ccf6e-f2ff-4ca0-92eb-a750e9a38435)